boards/toradex/verdin_am62/verdin_am62_am6234_m4.yaml

```diff
@@ -0,0 +1,8 @@
+identifier: verdin_am62/am6234/m4
+name: Toradex Verdin AM62
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+ram: 192
```

Comment:
`zephyr,sram = &sram0;` suggests the core has 1 MByte available, or am I wrong?

Reply:
And isn't the `256KB TCM` the primary SRAM for the M4F core?

Reply:
Actually the `sram0` comes from `<ti/am62x_m4.dtsi>`, where it is defined with 192 KB. And this, according to memory map, is correct, the first ITCM goes from 0x0 to 0x2FFFF.